windows/src/engine/kmcomapi/kmcomapi.cfg

```diff
@@ -1,46 +0,0 @@
--$A1
```

Comment:
I take it this .cfg file is no longer needed?

Reply:
Correct -- from earlier versions of Delphi. At some point we should remove all of them (#519)